--- blender-3.3.10/source/blender/blenfont/intern/blf_font_default.c.orig	2023-08-29 21:13:45.888397875 +0200
+++ blender-3.3.10/source/blender/blenfont/intern/blf_font_default.c	2023-08-29 21:25:10.372376410 +0200
@@ -18,7 +18,7 @@
 
 static int blf_load_font_default(const char *filename, const bool unique)
 {
-  const char *dir = BKE_appdir_folder_id(BLENDER_DATAFILES, BLF_DATAFILES_FONTS_DIR);
+  const char *dir = "/usr/share/fonts/blender";
   if (dir == NULL) {
     fprintf(stderr,
             "%s: 'fonts' data path not found for '%s', will not be able to display text\n",
