diff -Nur blender-creator-ph-2.25b.9.orig/intern/python/blendermodule/Makefile.am blender-creator-ph-2.25b.9/intern/python/blendermodule/Makefile.am
--- blender-creator-ph-2.25b.9.orig/intern/python/blendermodule/Makefile.am	Thu Oct 17 21:49:33 2002
+++ blender-creator-ph-2.25b.9/intern/python/blendermodule/Makefile.am	Tue Oct 22 00:19:31 2002
@@ -1,7 +1,8 @@
-bin_PROGRAMS = blendermodule
-#XXX: autofind path
+prefix = @PYTHONSYSPREFIX@
+exec_prefix = @PYTHONSYSPREFIX@
 INCLUDES = -I@PYTHONSYSPREFIX@/include/python@PYTHON_VERSION@
-blendermodule_SOURCES = main.c 
-blendermodule_LDADD   = -L@PYTHONSYSPREFIX@/lib/python@PYTHON_VERSION@/config -lpython@PYTHON_VERSION@ -ldl -lm -lpthread
-SUBDIRS = 
+pkgpyexec_LTLIBRARIES = Blendermodule.la
+Blendermodule_la_SOURCES = main.c 
+Blendermodule_la_LDFLAGS = -module -avoid-version -export-symbols-regex initBlender
+Blendermodule_la_LIBADD   = -L@PYTHONSYSPREFIX@/lib/python@PYTHON_VERSION@/config -lpython@PYTHON_VERSION@ -ldl -lm -lpthread
 EXTRA_DIST = main.c 
diff -Nur blender-creator-ph-2.25b.9.orig/intern/python/freeze/Makefile.am blender-creator-ph-2.25b.9/intern/python/freeze/Makefile.am
--- blender-creator-ph-2.25b.9.orig/intern/python/freeze/Makefile.am	Fri Oct 18 04:02:01 2002
+++ blender-creator-ph-2.25b.9/intern/python/freeze/Makefile.am	Mon Oct 21 22:25:27 2002
@@ -32,7 +32,7 @@
 
 ../frozen/libfrozen.a: $(OBJS)
 #	python -S -O freeze.py -d -x os -x pprint -x Blender -I ../modules -o ../frozen ../modules/VRMLmain.py
-	@PYTHON@@PYTHON_VERSION@ -S -O freeze.py -d -x os -x pprint -x Blender -I ../modules -o ../frozen ../modules/VRMLmain.py
+	@PYTHON@ -S -O freeze.py -d -x os -x pprint -x Blender -I ../modules -o ../frozen ../modules/VRMLmain.py
 #Or leave out @PYTHON@@PYTHON_VERSION@?
 	cd ../frozen && make -f Makefile.freeze && cd ../freeze
 
