--- blender-2.42a/source/gameengine/Expressions/ListValue.cpp.orig	2004-06-07 12:03:11.000000000 +0100
+++ blender-2.42a/source/gameengine/Expressions/ListValue.cpp	2006-09-29 14:34:38.000000000 +0100
@@ -23,12 +23,12 @@
 #include <config.h>
 #endif
 
-int listvalue_bufferlen(PyObject* list)
+Py_ssize_t listvalue_bufferlen(PyObject* list)
 {
 	return ( ((CListValue*)list)->GetCount());
 }
 
-PyObject* listvalue_buffer_item(PyObject* list,int index)
+PyObject* listvalue_buffer_item(PyObject* list,Py_ssize_t index)
 {
 	if (index >= 0 && index < ((CListValue*) list)->GetCount())
 	{
@@ -68,7 +68,7 @@
 
 
 /* just slice it into a python list... */
-PyObject* listvalue_buffer_slice(PyObject* list,int ilow, int ihigh)
+PyObject* listvalue_buffer_slice(PyObject* list,Py_ssize_t ilow, Py_ssize_t ihigh)
 {
 	int i, j;
 	PyListObject *newlist;
@@ -165,11 +165,11 @@
 static PySequenceMethods listvalue_as_sequence = {
 	(inquiry)listvalue_bufferlen,//(inquiry)buffer_length, /*sq_length*/
 	(binaryfunc)listvalue_buffer_concat, /*sq_concat*/
-	0,//(intargfunc)buffer_repeat, /*sq_repeat*/
-	(intargfunc)listvalue_buffer_item, /*sq_item*/
-	(intintargfunc)listvalue_buffer_slice, /*sq_slice*/
-	0,//(intobjargproc)buffer_ass_item, /*sq_ass_item*/
-	0,//(intintobjargproc)buffer_ass_slice, /*sq_ass_slice*/
+	(ssizeargfunc)0,//(intargfunc)buffer_repeat, /*sq_repeat*/
+	(ssizeargfunc)listvalue_buffer_item, /*sq_item*/
+	(ssizessizeargfunc)listvalue_buffer_slice, /*sq_slice*/
+	(ssizeobjargproc)0,//(intobjargproc)buffer_ass_item, /*sq_ass_item*/
+	(ssizessizeargfunc)0,//(intintobjargproc)buffer_ass_slice, /*sq_ass_slice*/
 };
 
 
@@ -178,7 +178,7 @@
 static PyMappingMethods instance_as_mapping = {
 	(inquiry)listvalue_bufferlen, /*mp_length*/
 	(binaryfunc)listvalue_mapping_subscript, /*mp_subscript*/
-	0,//(objobjargproc)instance_ass_subscript, /*mp_ass_subscript*/
+	(objobjargproc)0,//(objobjargproc)instance_ass_subscript, /*mp_ass_subscript*/
 };
 
 
