diff -durN blender2.20-linux-glibc2.1.2-i386.orig/plugins/bmake blender2.20-linux-glibc2.1.2-i386/plugins/bmake
--- blender2.20-linux-glibc2.1.2-i386.orig/plugins/bmake	Tue Aug 14 16:23:07 2001
+++ blender2.20-linux-glibc2.1.2-i386/plugins/bmake	Sat Sep  1 18:24:24 2001
@@ -5,36 +5,36 @@
 UNAME=`uname`
 if ( test $UNAME = "IRIX64" -o $UNAME = "IRIX" ) then
 	CC="cc";
-	CFLAGS="-KPIC -O2 -mips3";
+	CFLAGS="-KPIC -mips3";
 
 	LD="ld";
 	LDFLAGS="-shared -U";	
 elif ( test $UNAME = "FreeBSD" ) then
 	CC="gcc";
-	CFLAGS="-fPIC -funsigned-char -O2";
+	CFLAGS="-fPIC -funsigned-char";
 
 	LD="ld";
 	LDFLAGS="-Bshareable";
 elif ( test $UNAME = "OpenBSD" ) then
 	CC="gcc";
-	CFLAGS="-fPIC -funsigned-char -O2";
+	CFLAGS="-fPIC -funsigned-char";
 
 	LD="ld";
 	LDFLAGS="-Bshareable";
 elif ( test $UNAME = "Linux" ) then
 	CC="gcc";
-	CFLAGS="-fPIC -funsigned-char -O2";
+	CFLAGS="-fPIC -funsigned-char";
 
 	LD="ld";
 	LDFLAGS="-Bshareable";
 elif ( test $UNAME = "SunOS" ) then
 	CC="cc";
-	CFLAGS="-O2";
+	CFLAGS="";
 
 	LD="ld";
 	LDFLAGS="-r";
 fi
-
+CFLAGS="$CFLAGS $OPTFLAGS"
 if ( test "$#" = "1" ) then 
 	if ( test -f $1 ) then
 		BASE_FILE=`echo $1 | sed -e "1 s/\.c//g"`;
@@ -66,6 +66,8 @@
 	INCLUDES=-I..;
 elif ( test -f "../include/plugin.h" ) then
 	INCLUDES=-I../include
+elif ( test -f "/usr/X11R6/include/blender/plugin.h" ) then
+       INCLUDES=-I/usr/X11R6/include/blender
 else 
 	echo "Couldn't find plugin.h";
 	exit;
