--- blender-2.83.12/source/blender/blenfont/intern/blf_font_default.c~	2021-01-15 15:25:53.000000000 +0100
+++ blender-2.83.12/source/blender/blenfont/intern/blf_font_default.c	2021-03-06 19:32:29.170479890 +0100
@@ -33,7 +33,7 @@
 
 static int blf_load_font_default(const char *filename, const bool unique)
 {
-  const char *dir = BKE_appdir_folder_id(BLENDER_DATAFILES, "fonts");
+  const char *dir = "/usr/share/fonts/blender";
   if (dir == NULL) {
     fprintf(stderr,
             "%s: 'fonts' data path not found for '%s', will not be able to display text\n",
