--- blender-2.81a/source/blender/makesdna/DNA_gpencil_types.h~	2019-12-14 15:25:53.000000000 +0100
+++ blender-2.81a/source/blender/makesdna/DNA_gpencil_types.h	2019-12-14 15:25:55.905953908 +0100
@@ -223,7 +223,6 @@
 
   /** Vertex weight data. */
   struct MDeformVert *dvert;
-  void *_pad3;
 
   bGPDstroke_Runtime runtime;
 } bGPDstroke;
--- blender-2.81a/source/blender/makesdna/DNA_view3d_types.h~	2019-12-14 15:41:30.000000000 +0100
+++ blender-2.81a/source/blender/makesdna/DNA_view3d_types.h	2019-12-14 15:42:53.605920053 +0100
@@ -180,6 +180,7 @@
   float curvature_valley_factor;
 
   struct IDProperty *prop;
+  void *_pad99;
 } View3DShading;
 
 /** 3D Viewport Overlay settings. */
