--- blender-2.76/intern/cycles/util/util_simd.h.orig	2016-03-21 13:06:35.312841949 +0100
+++ blender-2.76/intern/cycles/util/util_simd.h	2016-03-21 13:07:38.880293791 +0100
@@ -229,7 +229,7 @@
   int r = 0; asm ("btr %1,%0" : "=r"(r) : "r"(i), "0"(v) : "flags"); return r;
 }
 
-#if defined(__KERNEL_64_BIT__) || defined(__APPLE__)
+#if (defined(__KERNEL_64_BIT__) && !defined(__ILP32__)) || defined(__APPLE__)
 __forceinline size_t __bsf(size_t v) {
   size_t r = 0; asm ("bsf %1,%0" : "=r"(r) : "r"(v)); return r;
 }
@@ -271,7 +271,7 @@
 #endif
 }
 
-#if defined(__KERNEL_64_BIT__) || defined(__APPLE__)
+#if (defined(__KERNEL_64_BIT__) && !defined(__ILP32__)) || defined(__APPLE__)
 __forceinline size_t bitscan(size_t v) {
 #if defined(__KERNEL_AVX2__)
 #if defined(__KERNEL_64_BIT__)
@@ -313,7 +313,7 @@
   return i;
 }
 
-#if defined(__KERNEL_64_BIT__) || defined(__APPLE__)
+#if (defined(__KERNEL_64_BIT__) && !defined(__ILP32__)) || defined(__APPLE__)
 __forceinline size_t __bscf(size_t& v) 
 {
   size_t i = bitscan(v);
